index.html

```diff
@@ -1 +1,14 @@
 Diana Carachure
+About
+Diana has 10 years of experience in the nonprofit sector. Diana has worked in both direct client services and administration roles.
+She holds bachelor's in psychology and is currently enrolled in Code the Dream's intro to programming class. 
+Experience 
+Advancement Specialist May 2021- present
+-contracts billing, fundraiser support, quality assurance, etc
+Data 
+-kkn;ek
+Lead Case Manager
+-ikjfeqj;
+Connect
+Email
+Github
```

Comment:
Looks like there's still placeholders here. Not really important for this assignment, but you'll probably want to fill it in before we get too much further along to make sure you have some real content for your site!